RecoEcal/EgammaCoreTools/interface/Utils.h

```diff
@@ -0,0 +1,17 @@
+#ifndef RecoEcal_EgammaCoreTools_interface_Utils_h
+#define RecoEcal_EgammaCoreTools_interface_Utils_h
+
+#include <CLHEP/Units/GlobalPhysicalConstants.h>
+
+
+template<typename T>
+T normalizePhi(T phi)
+{
+    constexpr T pi(M_PI);
+    constexpr T pi2(2*M_PI);
+    while (phi>pi) phi -= pi2;
+    while (phi<-pi) phi += pi2;
+    return phi;
+}
```

Comment:
This is redundant, see
https://github.com/cms-sw/cmssw/blob/c24591cbfc949a134c5161b1d389f98bbe717100/DataFormats/Math/interface/normalizedPhi.h#L6-L9
https://github.com/cms-sw/cmssw/blob/c24591cbfc949a134c5161b1d389f98bbe717100/DataFormats/Math/interface/deltaPhi.h#L13-L20

Reply:
Thank you! I was sure that it must have existed somewhere, just didn't know where.